### ISSUE TITLE
[Question] - How to add new loss functions

### ISSUE THREAD

**Issue by pedronahum** [pytorch/pytorch#2324]
Hi,

Apologies in advance if this is a silly question, but If I would like to play around with "new" lose functions, what would be steps needed to add them?

Thanks,

Pedro N.

**Comment by fmassa** [comment]
We use github issues to track bugs and feature requests.
Questions are better suited [for the forums](https://discuss.pytorch.org/).
BTW, there are a number of posts on the forum on how to create a new loss function, for example https://discuss.pytorch.org/t/build-your-own-loss-function-in-pytorch/235
Thanks!

**Comment by pedronahum** [comment]
Understood. Thank you
